components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/AssignUserRolesV2FunctionImpl.java

```diff
@@ -42,6 +42,7 @@
 public class AssignUserRolesV2FunctionImpl implements AssignUserRolesV2Function {
 
     private static final Log LOG = LogFactory.getLog(AssignUserRolesV2FunctionImpl.class);
+    private static final String INTERNAL_DOMAIN_PREFIX = "internal/";
```

Comment:
use UserCoreConstants.INTERNAL_DOMAIN

Reply:
Used UserCoreUtil to remove internal domain